src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -117,16 +117,14 @@ interface ISectionProps extends ICaptionProps {
   imageAlt?: string;
   imageHeight?: number;
   imageWidth?: number;
-  imageIsSquare?: boolean;
 }
 
 export const Section: React.FC<ISectionProps> = props => {
   if (props.imageSource) {
     const imageStyles = {
       width: props.imageWidth,
       height: props.imageHeight,
-      maxWidth: props.imageIsSquare ? 160 : undefined,
```

Comment:
I think you'll still want the `imageIsSquare`, but set to `320`, in order to prevent image skew from a `height` set that conflicts with a `maxWidth`. Or am I missing something?

Reply:
Oh, good spot. This update may be ill advised. I didn't see any images with width or heights explicitly set so the thinking here was that the image would proportionally fill the div up to 320px wide. Perhaps a more appropriate solve would be to set the height and width explicitly but I think images with details won't work below 160px. 

Either way I think we're seeing that this does indeed need more investigation so I'll can this one in favor of design investigation.